sim/SPI_v3/components/Loopback.py

```diff
@@ -0,0 +1,51 @@
+'''
+==========================================================================
+Loopback.py
+==========================================================================
+This takes receives a packet from the SPIMinionAdapter module and sends it back the next cycle to the 
+SPIMinionAdapter module. Uses val/rdy microprotocol.
+
+Author: Dilan Lakhani
+    December 13, 2021
+
+'''
+
+from pymtl3 import *
+from pymtl3.stdlib.basic_rtl import RegisterFile, Mux, RegEnRst, RegEn, Adder, Incrementer, LeftLogicalShifter
+from pymtl3.stdlib.stream.ifcs import RecvIfcRTL, SendIfcRTL
+
+
+class Loopback( Component ):
+    
+    def construct( s, nbits=32):
```

Comment:
Let's be consistent and stick to a 2-space indention

Reply:
Indentation changed to 2-space